probe/probe.go

```diff
@@ -81,23 +83,40 @@ func PrintProbeResult(c *k8s.Client, o Options) error {
 		}
 	}
 	if isSystemdMode() {
-		err := probeSystemdMode()
+		err := probeSystemdMode(o)
 		if err != nil {
 			return err
 		}
 		return nil
 	}
-	if isKubeArmorRunning(c, o) {
-		getKubeArmorDeployments(c, o)
-		getKubeArmorContainers(c, o)
-		err := probeRunningKubeArmorNodes(c, o)
+	isRunning, daemonsetStatus := isKubeArmorRunning(c, o)
+	if isRunning {
+		deploymentData := getKubeArmorDeployments(c, o)
+		containerData := getKubeArmorContainers(c, o)
+		nodeData, postureData, err := probeRunningKubeArmorNodes(c, o)
 		if err != nil {
```

Comment:
Should we continue if an error occurs?

Reply:
Imo, we should continue to print the output from other functions. wdyt?